### PR TITLE
Add basic UI tests, testing drawer behavior

### PR DESCRIPTION
**Pull request by tiembo**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -79,6 +79,22 @@ dependencies {
 
     implementation "com.crashlytics.sdk.android:crashlytics:${versions.crashlytics}"
     implementation "com.google.firebase:firebase-core:${versions.firebase}"
+
+    testImplementation project(':test_shared')
+    testImplementation "android.arch.core:core-testing:${versions.lifecycle}"
+    testImplementation "com.nhaarman:mockito-kotlin:${versions.mockito_kotlin}"
+    testImplementation "junit:junit:${versions.junit}"
```
Comment (ataulm) [inline app/build.gradle]: what does this block of dependencies help us do?

File: app/build.gradle [hunk 2]
```diff
@@ -79,6 +79,22 @@ dependencies {
...
+    androidTestImplementation project(':test_shared')
+    androidTestImplementation "android.arch.core:core-testing:${versions.lifecycle}"
+    androidTestImplementation "com.android.support.test:runner:${versions.test_runner}"
+    androidTestImplementation "com.android.support.test:rules:${versions.test_rules}"
+    androidTestImplementation "com.android.support.test.espresso:espresso-contrib:${versions.espresso}"
+    androidTestImplementation "com.android.support.test.espresso:espresso-core:${versions.espresso}"
+    androidTestImplementation "com.android.support.test.uiautomator:uiautomator-v18:${versions.ui_automator}"
+    androidTestImplementation "com.nhaarman:mockito-kotlin:${versions.mockito_kotlin}"
+    androidTestImplementation "org.mockito:mockito-android:${versions.mockito}"
+    androidTestImplementation "org.mockito:mockito-core:${versions.mockito}"
```
Comment (ataulm) [inline app/build.gradle]: Do we need mockito as an androidTest dependency?

File: app/build.gradle (hunk 1 above)
Comment (tiembo) [reply app/build.gradle]: This block was copied over from another module.  Pruned this list down to only the required dependencies.

File: app/build.gradle (hunk 2 above)
Comment (tiembo) [reply app/build.gradle]: Nah - pruned down the list.

**Comment by tiembo** [comment]
Thanks for the super fast review @ataulm!

**Pull request merged**